src/bin-utils/index.test.js

```diff
@@ -305,6 +311,69 @@ test(
   },
 )
 
+test('specificHelpScript: returns no script available', () => {
+  const config = {scripts: {}}
+  const message = specificHelpScript(config, 'foo')
+  const expected = chalk.yellow('Script foo does not exist')
+  expect(message).toBe(expected)
+})
+
+test(
+  'specificHelpScript: do not display script with flag hiddenFromHelp=true',
+  () => {
+    const config = {
+      scripts: {
+        foo: {
+          description: 'the foo script',
+          script: 'echo "foo"',
+          hiddenFromHelp: true,
+        },
+      },
+    }
+    const message = specificHelpScript(config, 'foo')
+    const expected = chalk.yellow('Script foo does not exist')
+    expect(message).toBe(expected)
+  },
+)
+
+test('specificHelpScript: formats a nice message', () => {
+  const config = {
```

Comment:
To make sure that you can handle nesting and prefixes, maybe change this config to:

```javascript
const config = {
  scripts: {
    foo: {
      bar: {
         description: 'the bar script',
          script: 'echo "bar"',
      }
    }
  }
}
```

And do:

```js
const message = specificHelpScript(config, 'fo.ba')
```